src/components/02_atoms/Widgets/EntityReferenceAutocompleteWidget.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import keycode from 'keycode';
+import Downshift from 'downshift';
+
+import Paper from '@material-ui/core/Paper';
+import TextField from '@material-ui/core/TextField';
+import Chip from '@material-ui/core/Chip';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import WidgetPropTypes from '../../05_pages/NodeForm/WidgetPropTypes';
+
+import api from './../../../utils/api/api';
+
+class EntityReferenceAutocompleteWidget extends React.Component {
```

Comment:
We probably shouldn't include `Widget` in the name since most of the widgets doesn't have it.

Reply:
Sure, let's do that!

Reply:
Do you want to put up a PR for the file upload widget?